README.md

```diff
@@ -68,6 +68,8 @@ If you are still using the legacy [Access scopes][access-scopes], the `https://w
 | `web.listen-address`<br />`STACKDRIVER_EXPORTER_WEB_LISTEN_ADDRESS` | No | `:9255` | Address to listen on for web interface and telemetry |
 | `web.telemetry-path`<br />`STACKDRIVER_EXPORTER_WEB_TELEMETRY_PATH` | No | `/metrics` | Path under which to expose Prometheus metrics |
 
+__NOTE:__ monitoring.metrics-interval is not the scape interval; it specifies the metric's timestamp interval to send in the request to the Stackdriver API.
```

Comment:
More than adding a note here, I'd prefer to update the `monitoring.metrics-interval` field description, so instead of being `Interval to request the Google Stackdriver Monitoring Metrics for. Only the most recent data point is used` it might be `Metric's timestamp Interval to request from the Google Stackdriver Monitoring Metrics API. Only the most recent data point is used` (or something like that).